SUMMARY.md

```diff
@@ -33,6 +33,7 @@
   * [Addresses](operations/addresses.md)
   * [Age categories](operations/agecategories.md)
   * [Availability blocks](operations/availabilityblocks.md)
+  * [Availability adjustments](operations/availabilityadjustments.md)
```

Comment:
```suggestion
  * [Availability adjustments](operations/availabilityadjustments.md)
  * [Availability blocks](operations/availabilityblocks.md)
```

I think we are using alphabetic ordering 🤔

Reply:
yes we are :-)